src/frontend/Sider/Behandling/SettPåVent/SettPåVentContainer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { AGray100 } from '@navikt/ds-tokens/dist/tokens';
+
+import SettPåVentForm from './SettPåVentForm';
+import SettOpVentInformasjon from './SettPåVentInformasjon';
+import { StatusSettPåVent } from './typer';
+import { useApp } from '../../../context/AppContext';
+import { useBehandling } from '../../../context/BehandlingContext';
+import DataViewer from '../../../komponenter/DataViewer';
+import { BehandlingStatus } from '../../../typer/behandling/behandlingStatus';
+import { byggTomRessurs, Ressurs, RessursStatus } from '../../../typer/ressurs';
+
+const Container = styled.div`
+    margin: 2rem;
+    padding: 2rem;
+    background: ${AGray100};
+`;
+
+const SettPåVentContainer = () => {
+    const { request } = useApp();
+    const { behandling, statusPåVentRedigering } = useBehandling();
+
+    const [statusResponse, settStatusResponse] =
+        useState<Ressurs<StatusSettPåVent>>(byggTomRessurs());
+
+    useEffect(() => {
+        if (behandling.status === BehandlingStatus.SATT_PÅ_VENT) {
+            request<StatusSettPåVent, null>(`/api/sak/sett-pa-vent/${behandling.id}`).then(
+                settStatusResponse
+            );
+        }
+        // skal kun hente status vid første rendering, ellers håndteres det av komponenten selv
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
Vil ikke å ha "behandling.id" her kun trigge rerendring dersom den faktisk endrer seg som er det samme som når den lastes inn?

Reply:
Jo, samtidig vil det egentlige ikke skje. 
Her burde man nesten ha en egen håndtering av det lengre ut for at allt av domtreet blir oppdatert hvis behandling endrer seg. Eks ved å sette key=behandling.id langt ut